sys/net/gnrc/transport_layer/tcp/gnrc_tcp_common.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2020 Simon Brummer
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_gnrc
+ * @{
+ *
+ * @file
+ * @brief       GNRC TCP common function implementation
+ *
+ * @author      Simon Brummer <simon.brummer@posteo.de>
+ * @}
+ */
+
+#include "internal/common.h"
+
+tcb_list_t *_gnrc_tcp_common_get_tcb_list(void)
+{
+    static tcb_list_t list = {NULL, MUTEX_INIT};
+    return &list;
```

Comment:
I think this should rather go to global scale. While this is doing effectively the same, I saw cases were it did not. This also makes it easier to debug, as you can put a watch-point on `gnrc_tcp_common.c:_list` from anywhere this way.

```suggestion

static tcb_list_t _list = {NULL, MUTEX_INIT};

tcb_list_t *_gnrc_tcp_common_get_tcb_list(void)
{
    return &_list;
```

Reply:
I'll move it outside of the function, for the sake of the debugging argument.

Out of curiosity, do you have any example were both versions don't produce the same results?